packages/client/tests/memory/_utils/runMemoryTest.ts

```diff
@@ -0,0 +1,135 @@
+import chalk from 'chalk'
+import childProcess from 'child_process'
+import { once } from 'events'
+import fs from 'fs/promises'
+// import { byline } from '@prisma/internals'
+import { linearRegression } from 'simple-statistics'
+
+import { buildMemoryTest } from './buildMemoryTest'
+import { dropMemoryTestDatabase, setupMemoryTestDatabase } from './database'
+import { generateMemoryTestClient } from './generateMemoryTestClient'
+import { generateMemoryUsageReport } from './generateMemoryUsageReport'
+import { MemoryTestDir } from './MemoryTestDir'
+
+const GROWTH_RATE_THRESHOLD_IN_BYTES = 200
```

Comment:
What made you pick 200?

Reply:
Just a bunch of runs with leaking vs non-leaking client. Although, I think, I've picked up the threshold before I've added v8 flags for disabling caches and optimizations, now memory consumption is much stable from iteration to iteration and this threshold can be lowered.

Reply:
Lowered it to 50, since after latest changes growth rate stays close to 0.